server/eclipse-project/src/main/java/us/freeandfair/corla/json/CountyDashboardRefreshResponse.java

```diff
@@ -72,12 +72,22 @@
   /**
    * The ballot manifest digest.
    */
-  private final String my_ballot_manifest_digest;
+  private final String my_manifest_digest;
+  
+  /**
+   * The ballot manifest timestamp.
+   */
+  private final Instant my_manifest_timestamp;
```

Comment:
**Not** for this PR, but: I'd like it if we changed the serialization of these values to match the normal JSON serialization of date times, instead of being a serialized `Instant`. In other words:
```js
> JSON.stringify(new Date())
'"2017-08-17T17:35:31.614Z"'
```

I'll make an issue for it for phase 2.